src/Generators/TestsGenerator.php

```diff
@@ -41,7 +41,8 @@ protected function generateTests(): void
             'databaseTableName' => $this->getTableName($this->model),
             'entities' => $this->getTableName($this->model, '-'),
             'withAuth' => $this->withAuth,
-            'modelsNamespace' => $this->getOrCreateNamespace('models'),
+            'modelsNamespace' => $this->getOrCreateNamespace('models', $this->modelSubFolder),
```

Comment:
let's make small refactoring in stub as well

```suggestion
            'entityNamespace' => $this->getOrCreateNamespace('models', $this->modelSubFolder),
```